dive_core/capture_data.cpp

```diff
@@ -348,10 +349,10 @@ const MemoryAllocationInfo &MemoryManager::GetMemoryAllocationInfo() const
 }
 
 //--------------------------------------------------------------------------------------------------
-bool MemoryManager::CopyMemory(void    *buffer_ptr,
-                               uint32_t submit_index,
-                               uint64_t va_addr,
-                               uint64_t size) const
+bool MemoryManager::CopyMem(void    *buffer_ptr,
```

Comment:
I feel that this func copies data from managed memory blocks to a provided buffer, with checks for submit indices and virtual addresses. Maybe `RetrieveMemoryData` is better? It suggests that data is being fetched from a managed source. (When i see CopyMemory, my first impression is to look for source and destination which are not in the args)